app/gunicorn.conf.py

```diff
@@ -0,0 +1,22 @@
+"""
+Configuration file for the Gunicorn server used to run the application in production environments.
+
+Attributes:
+    bind(str): The socket to bind. Formatted as '0.0.0.0:$PORT'.
+    threads(int): The number of threads per worker for handling requests.
+
+For more information, see https://docs.gunicorn.org/en/stable/configure.html
+"""
+
+from src.app_config import AppConfig
+
+app_config = AppConfig()
+
+bind = app_config.host + ':' + str(app_config.port)
+# Calculates the number of usable cores and doubles it. Recommended number of workers per core is two.
+# https://docs.gunicorn.org/en/latest/design.html#how-many-workers
+# We use 'os.sched_getaffinity(pid)' not 'os.cpu_count()' because it returns only allowable CPUs.
+# os.sched_getaffinity(pid): Return the set of CPUs the process with PID pid is restricted to.
+# os.cpu_count(): Return the number of CPUs in the system.
+threads = 4
+workers = 4
```

Comment:
What's the reason to use `4`?
https://github.com/navapbc/template-application-flask/blob/dded29ce1ae0d3b9cd067a971850a290da29e0c2/app/gunicorn.conf.py#L24 calculates it.

Reply:
I had an issue where os didn't have `'sched_getaffinity'` defined- this might have been another limitation of the system I was using. It seems to run without issue on the Docker container, so I'll add it here.